app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,41 @@
+<% content_for :head do %>
+  <%= stylesheet_link_tag 'registration', media: 'all' %>
+<% end %>
+<div class="main">
+<div class="submain">
+<h2>Sign up</h2>
+
+<%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
+  <%= render "devise/shared/error_messages", resource: resource %>
+ 
+ <div class="field name-label-input-container">
+  <%= f.label :name %><br />
+  <%= f.text_field :name, autofocus: true, autocomplete: "name", class:'name-input' %>
+</div>
+
+  <div class="field">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password %>
+    <% if @minimum_password_length %>
+    <em>(<%= @minimum_password_length %> characters minimum)</em>
+    <% end %><br />
+    <%= f.password_field :password, autocomplete: "new-password" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password_confirmation %><br />
+    <%= f.password_field :password_confirmation, autocomplete: "new-password" %>
+  </div>
+
+  <div class="actions">
+    <%= f.submit "Sign up" %>
+  </div>
+<% end %>
```

Comment:
- [ Required ] I think it would be better if you style the `sign up` form the same way it is on the project wireframe. Currently, your design differs from the project wireframe in the following ways.
    
  - [ ] The width of the signup `button` should be the same as the width of other `input` elements in the form and should fill the width of the form.
 

## Your design
![Screenshot from 2023-08-04 23-14-27](https://github.com/massazoum/Rails_App/assets/16392046/5f0747a4-c375-4591-86aa-d5a60fe7adee)

## Project wireframe
![](https://github.com/microverseinc/curriculum-rails/raw/main/blog-app/images/blog_login.png)